iOS/MSMapsModules/MSMapsModules/MSGeoJsonParser/MSGeoJsonParser.h

```diff
@@ -0,0 +1,22 @@
+//
+//  MSGeoJsonParser.h
+//  MSMapsModules
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/12/20.
+//  Copyright © 2020 Microsoft.
+//  Licensed under the MIT license.
+//
+
+#import <MicrosoftMaps/MicrosoftMaps.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSGeoJsonParser : NSObject
+
+- (MSGeoJsonParser *)init NS_UNAVAILABLE;
```

Comment:
I've noticed that in our existing code base, we would use MSGeoJsonParser* instead of MSGeoJsonParser *. I'm not sure if we want this project to follow suit or not. Maybe something you should look into for more input

Reply:
I believe this was run through clang-format with default settings. Can you add the format script to this PR as well?

Reply:
Script added.